test/lib/ansible_test/_data/completion/remote.txt

```diff
@@ -3,6 +3,7 @@ freebsd/12.1 python=3.6,2.7 python_dir=/usr/local/bin
 freebsd/12.2 python=3.7,2.7 python_dir=/usr/local/bin
 osx/10.11 python=2.7 python_dir=/usr/local/bin
 macos/10.15 python=3.8 python_dir=/usr/local/bin
+macos/11.0.1 python=3.8 python_dir=/usr/local/bin
```

Comment:
Perhaps we should use Python 3.9 on macOS 11 since it's the latest release available?

Reply:
I'm fine with that.

Reply:
I updated the guest image with Python 3.9 and submitted a PR to `ansible-core-ci`.